arroyo-console/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import {
+  Button,
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react';
+import { TourContext, TourSteps } from '../tour';
+
+const WelcomeModal: React.FC = () => {
+  const { tourStep, setTourStep, disableTour } = useContext(TourContext);
+
+  return (
+    <Modal
+      isOpen={tourStep == TourSteps.WelcomeModal}
+      onClose={() => {}}
+      isCentered
+      size={'xl'}
+      variant={'tour'}
+    >
+      <ModalOverlay />
+      <ModalContent padding={2}>
+        <ModalCloseButton onClick={disableTour} />
+        <ModalBody display={'flex'} gap={3} flexDirection={'column'} alignItems={'center'}>
+          <img width="320" src="/logo.svg" />
+          <Text fontSize={'lg'} fontWeight={'bold'} textAlign={'center'}>
+            Welcome!
+          </Text>
+          <Text fontSize={'md'} textAlign={'center'}>
```

Comment:
nit: left align

Reply:
How's this? I also moved the "Welcome!" into the body.
![image](https://github.com/ArroyoSystems/arroyo/assets/8881183/3c9695da-8b71-4283-bfac-0b2683059efc)

Reply:
Looks good!